src/js/components/molecules/tooltip-toggle/tooltip-toggle.scss

```diff
@@ -1,10 +1,27 @@
 .tooltip-toggle {
     position: relative;
-    display: inline-block;
```

Comment:
I think it is an inline-block element like a button. It shoudn't take the 100% width of the containing element.